integration/kubernetes/cleanup_bare_metal_env.sh

```diff
@@ -11,6 +11,8 @@ set -o pipefail
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source "${SCRIPT_PATH}/../../lib/common.bash"
 
+info "Clean up bare metal env"
+for_retry="${1:-false}"
```

Comment:
Is it better to change `for_retry` to `keep_cni_bin` for understanding?

Reply:
Yep, You are right. It would deliver the clearer meaning. Thanks for pointing out. It has been reflected. :heart: